src/staging/src/templates/park.js

```diff
@@ -354,7 +354,7 @@ export default function ParkTemplate({ data }) {
               sm={12}
               md={9}
               lg={9}
-              className={classes.parkContent}
+              className={`${classes.parkContent} main-content`}
```

Comment:
I tried this, and I think it doesn't work. Try running `Gatsby serve` to run it locally from a full build, and check to see if it works.